content/plugins/aws.md

```diff
@@ -228,6 +228,59 @@ See the [common Runtime Properties section](#runtime-properties).
 
 Note that the actual IP is available via the `aws_resource_id` runtime-property.
 
+## cloudify.aws.nodes.Volume
+
+**Derived From:** [cloudify.nodes.Volume]({{< relref "blueprints/built-in-types.md" >}})
+
+**Properties:**
+
+  * `size` This is the size in GB.
+  * `zone` An AWS availability zone, such as `us-east-1b`.
+  * `device` A device on the attached instance, such as `/dev/xvdf`. Note that this needs to be a valid device name on the OS.
```

Comment:
~such as~ __for example,__  `/dev/xvdf`. Note that this ~needs to~ __must__ be a valid device name on the OS.